pep-0579.rst

```diff
@@ -2,13 +2,29 @@ PEP: 579
 Title: Refactoring C functions and methods
 Author: Jeroen Demeyer <J.Demeyer@UGent.be>
 BDFL-Delegate: Petr Viktorin
-Status: Draft
+Status: Accepted
```

Comment:
Informational PEPs should either be "active" because they are a living document or "final" because you're done making changes.